coverage.py

```diff
@@ -717,7 +717,12 @@ class CollectionCoverageProvider(IdentifierCoverageProvider):
     # CoverageProvider. If this CoverageProvider can manage collections
     # for any protocol, leave this as None.
     PROTOCOL = None
-    
+
+    # By default, Works calculated by a CollectionCoverageProvider update
+    # the ExternalSearchIndex. Set this value to True for applications that
+    # use external search, such as the Metadata Wrangler.
+    EXCLUDE_SEARCH_INDEX = False
```

Comment:
The layers of negatives here confuse me. 1) Why do we want to exclude applications that use external search, instead of applications that don't use it? 2) I thought the Metadata Wrangler _doesn't_ use external search.